Libs/MRML/Core/vtkMRMLModelDisplayNode.h

```diff
@@ -74,7 +75,20 @@ class VTK_MRML_EXPORT vtkMRMLModelDisplayNode : public vtkMRMLDisplayNode
   virtual void SetActiveAttributeLocation(int location);
 
   /// Update the pipeline based on this node attributes
-  virtual void UpdatePolyDataPipeline();
+  virtual void UpdateMeshPipeline();
+
+  vtkGetMacro(Threshold,bool);
+  vtkSetMacro(Threshold,bool);
+  vtkBooleanMacro(Threshold,bool);
+
+  /// Set the threshold range of the model display node.
+  /// \sa GetThresholdMin(), GetTresholdMax()
+  void SetThresholdRange(double min, double max);
```

Comment:
for the `GetThresholdRange` with double[2] output, the values min and max are retrieved from `this->ThresholdFilter` directly so I can't return a double* to the double[2] range since it would be a local variable. Would the signature `GetThresholdRange(double range[2])` be alright? The user would need to take care of instantiating the double[2] and pass it as an argument.